test/Streamly/Test/Network/Inet/TCP.hs

```diff
@@ -135,10 +135,15 @@ moduleName :: String
 moduleName = "Network.Inet.TCP"
 
 main :: IO ()
-main = hspec $ do
+main =
+#if !defined(mingw32_HOST_OS)
+    hspec $ do
     modifyMaxSuccess (const 1) $ do
       describe moduleName $ do
         describe "Accept Connections" $ do
             prop "acceptOnPort" validateOnPort
             prop "acceptOnPortLocal" validateOnPortLocal
         --  prop "acceptOnAddr/connect" Tested as part of above test cases
+#else
```

Comment:
I thought there are only 1 or 2 tests failing, and there are many others that are not failing.

Since all tests are failing, it makes sense to disable `-threaded` for windows case so that we at least test something.